### PR TITLE
Adding HappyBase Batch.put() and helpers.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: gcloud/bigtable/happybase/batch.py [hunk 1]
```diff
@@ -133,3 +191,54 @@ def __exit__(self, exc_type, exc_value, traceback):
         # NOTE: For non-transactional batches, this will even commit mutations
```
Comment (jonparrott) [inline gcloud/bigtable/happybase/batch.py]: Would this be more helpful to the user as part of the docstring?

**Comment by jonparrott** [comment]
LGTM, with slight nit

File: gcloud/bigtable/happybase/batch.py (hunk 1 above)
Comment (dhermes) [inline gcloud/bigtable/happybase/batch.py]: Is the docstring for `__enter__` ever surface anywhere?

Comment (jonparrott) [inline gcloud/bigtable/happybase/batch.py]: :(

Is this behavior considered surprising? If so, I feel it should be documented somewhere that's visible.

Comment (dhermes) [inline gcloud/bigtable/happybase/batch.py]: 1. See the latest commit.
1. This code wasn't even in this PR, no fair!
1. The behavior is in place to [match the HappyBase][1] implementation.

[1]: https://github.com/wbolster/happybase/blob/9cbd718c10a3089f234f1eac1236b631e1f8e7cd/happybase/batch.py#L125

**Comment by dhermes** [comment]
@jonparrott I'm going to merge now (a HappyBase `table` helper relies on `_get_column_pairs`). If not note in the `Batch` docstring is insufficient LMK and I can iterate on it.

**Pull request merged**